src/cockpit/389-console/src/lib/replication/replSuffix.jsx

```diff
@@ -231,24 +232,23 @@ export class ReplSuffix extends React.Component {
         });
 
         this.props.disableTree();
-        log_cmd('enableReplication', 'Enable replication', cmd);
-        cockpit
-                .spawn(cmd, { superuser: true, err: "message" })
-                .done(content => {
-                    this.props.reload(1);
-                    this.props.addNotification(
-                        "success",
-                        `Successfully enabled replication for "${this.props.suffix}"`
-                    );
-                })
-                .fail(err => {
-                    this.props.reload(1);
-                    const errMsg = JSON.parse(err);
-                    this.props.addNotification(
-                        "error",
-                        `Failed to enable replication for "${this.props.suffix}" - ${errMsg.desc}`
-                    );
-                });
+
+
+        // Something changed, perform the update
+        const config = {
+            cmd: cmd,
+            promptArg: "--bind-passwd-prompt",
+            passwd: passwd,
+            addNotification: this.props.addNotification,
+            success_msg: `Successfully enabled replication for "${this.props.suffix}"`,
+            error_msg: `Failed to enable replication for "${this.props.suffix}"`,
```

Comment:
I'm not sure why but enable replication fails for me with the next error:

```
Danger alert:
Failed to enable replication for "dc=example,dc=com": usage: dsconf [-h] [-v] [-D BINDDN] [-w BINDPW] [-W] [-y PWDFILE] [-b BASEDN]
              [-Z] [-j]
              instance
              {backend,backup,chaining,config,directory_manager,monitor,plugin,pwpolicy,localpwp,replication,repl,repl-agmt,repl-winsync-agmt,repl-tasks,sasl,security,schema,repl-conflict}
              ...
dsconf: error: unrecognized arguments: undefined
```

The steps to reproduce:
[Screencast from 03-03-2023 08:11:10 AM.webm](https://user-images.githubusercontent.com/5634892/222770561-4fc21648-1f45-4a73-81ce-d391f4bd025f.webm)

Reply:
Fixing this now...

Reply:
@droideck - should be fixed now...